judge/migrations/0125_infer_private_bools.py

```diff
@@ -0,0 +1,29 @@
+# Generated by hand on 2021-12-15
+
+from django.db import migrations
+
+
+def sync_private_booleans(apps, schema_editor):
+    Problem = apps.get_model('judge', 'Problem')
+    Contest = apps.get_model('judge', 'Contest')
+    db_alias = schema_editor.connection.alias
+
+    for problem in Problem.objects.using(db_alias).all():
+        problem.is_organization_private = problem.organizations.exists()
```

Comment:
Should we do this instead?
 - If `is_organization_private` is false, remove all organizations
 - If `is_organization_private` is true and there are no organizations, set `is_organization_private` to false

There are quite a few problems that have `is_organization_private` set to false, but have a non-empty `organizations` field. Of the top of my head, most of the WCIPEG migrated problems still have `WCIPEG Migration` in `organizations`. If we keep this code as-is, all of those problems will be set to organization-private, and we need to track down which ones to make public.